### PR TITLE
savanna depends on adios+sz

### PR DESCRIPTION
**Pull request by kshitij-v-mehta**
Explicitly adding variant sz to adios since it is no longer enabled by default in adios

### CODE REVIEW

**Comment by adamjstewart** [comment]
Oh, I assume you meant the `+sz` variant that will be added once #4857 is merged. We should wait for that PR to be merged before we merge this.

**Comment by kshitij-v-mehta** [comment]
Yes, sorry I should have mentioned that in the PR description.

File: var/spack/repos/builtin/packages/savanna/package.py [hunk 1]
```diff
@@ -43,7 +43,7 @@ class Savanna(MakefilePackage):
 
     depends_on('mpi')
     depends_on('stc')
-    depends_on('adios +fortran +sz +staging')
+    depends_on('adios +fortran +staging')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/savanna/package.py]: Please do not remove this change. Packages should **never** rely on default variants. The default may change from time-to-time, and users may override the defaults in `packages.yaml` or on the command-line. If `savanna` only works with `adios+sz`, you have to make that explicit.

**Comment by ax3l** [comment]
Also: please do not make sz enabled by default yet in ADIOS itself, since it will break the adios package as I wrote in both PRs :)

**Comment by kshitij-v-mehta** [comment]
> Also: please do not make sz enabled by default yet in ADIOS itself, since it will break the adios package as I wrote in both PRs :)

I am going to update the sz package (I am not the original owner) to have only the develop version as the other version does not work

**Comment by ax3l** [comment]
Keep the  sz package as is and write in savanna :

+sz@develop

**Comment by adamjstewart** [comment]
Wait a minute, I was able to install `sz@1.4.9.2` without any trouble. Have you tried that version yet?

**Comment by kshitij-v-mehta** [comment]
> I was able to install sz@1.4.9.2 without any trouble. Have you tried that version yet?

The sz package has `version('1.4.9-beta', '8a00cddc39780c18d766215f69713f82')` , which seems to be wrong. I haven't tried v1.4.9.2 yet.

**Comment by adamjstewart** [comment]
Yes, it looks like the checksum for that version has changed. Try downloading straight from GitHub, it worked fine for me.

**Comment by adamjstewart** [comment]
Then we won't need to argue over `sz` vs `sz@develop` 😆

**Pull request merged**